.github/workflows/build-docker.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
           node-version: '16.13'
 
       - name: Ignore Husky
-        run: npm set-script prepare ""
+        run: npm pkg delete scripts.prepare
```

Comment:
`set-script` won't work in node 18 when we switch, so pre-emptively changing it here